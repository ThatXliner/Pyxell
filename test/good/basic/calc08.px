
print 2.8 - 5 + 3.13
print 9 * 6.4, (2 + 11.5) / 3
print 2^-0.5^-1, 10^20
