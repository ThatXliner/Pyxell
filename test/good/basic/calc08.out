0.93
57.6 4.5
0.25 1e+20
